frontend/src/components/SecondaryInfo.vue

```diff
@@ -28,6 +28,20 @@ const state = reactive({
   awayShotsOnGoal: 24
 })
 
+async function fetchShots() {

```

Comment:
I'd like to see a reduction of repeated code between this and PrimaryInfo. I'm not to familiar with optimal code organization with ts and vue, but I'm certain there is a better way.

Reply:
Agreed with this. We'll want to have some kind of common library that can handle all of our `fetch` calls.